web/src/package.json

```diff
@@ -132,6 +132,7 @@
     "eslint-config-standard-jsx": "^6.0.2",
     "eslint-plugin-cypress": "^2.1.2",
     "eslint-plugin-import": "2.17.3",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
```

Comment:
We are starting to use explicit versioning in our `package.json` as we've running into issues with our `yarn.lock` file with duplicate declarations of packages if there are peer dependency conflicts. You can fix this pretty easily by removing the `^` and rerunning `yarn install`